app/controllers/user.rb

```diff
@@ -0,0 +1,35 @@
+get '/users/new' do
+  @user = User.new
+  # if @user.valid?
+  #   redirect '/'
+  # else
+  #   status 422
+  #   @errors = user.errors.full_messages
+  erb :'users/new'
+  # end
+end
+
+get '/users/:id' do
+  @user = User.find_by(id: params[:id])
+  @suggestions = @user.suggestions.sort {|a,b| b.votes <=> a.votes}
```

Comment:
There is a SQL way to do this (and thus ActiveRecord way to do this), bonus for anyone who figures out how.